internal/pkg/service/buffer/api/service/service.go

```diff
@@ -40,6 +44,9 @@ func (*service) HealthCheck(dependencies.ForPublicRequest) (res string, err erro
 	return "OK", nil
 }
 
+// nolint:godox
+// TODO: collect errors instead of bailing on the first one
```

Comment:
Potřeboval bych poradit. Při create receiveru se může pokazit hodně věcí a bylo by lepší UX ty errory vyhodit všechny najednou, místo toho, aby user musil posílat deset requestů než dojde na všechny problémy. Nevím ale, jak na to.

Reply:
```
errs := errors.NewMultiError()
if err := abc(); err != nil {
  errs.Append(err)
}
return errs.ErrorOrNil()
```